### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by LiCern** [fac20/week10-be#23]
Lighthouse report is 96, good!

- [x] However, screen readers cannot access your landing page at all (it doesn't read anything) and cannot read the buttons on the fish fight page.

- [x] It is not possible to navigate the site using a keyboard at all unfortunately 🥺
